overlord/configstate/configcore/backlight_test.go

```diff
@@ -41,15 +41,10 @@ func (s *backlightSuite) SetUpTest(c *C) {
 
 	s.systemctlArgs = nil
```

Comment:
This is also done in configcoreSuite.SetUpTest() so this line can be removed too